docs/kubernetes-reliability.md

```diff
@@ -0,0 +1,101 @@
+Distributed system such as Kubernetes are designed to be resilient to the
```

Comment:
Make a document header for the first section, as well as a doc title. `# Overview` perhaps?